plugins/discovery-ec2/qa/amazon-ec2/build.gradle

```diff
@@ -48,28 +47,20 @@ processTestResources {
     MavenFilteringHack.filter(it, expansions)
 }
 
-integTestCluster {
-    dependsOn ec2Fixture
-    numNodes = ec2NumberOfNodes
-    plugin ':plugins:discovery-ec2'
-    keystoreSetting 'discovery.ec2.access_key', 'ec2_integration_test_access_key'
-    keystoreSetting 'discovery.ec2.secret_key', 'ec2_integration_test_secret_key'
+integTest {
+    dependsOn ec2Fixture, project(':plugins:discovery-ec2').bundlePlugin
```

Comment:
I don't think we should add cross project task dependencies here. While I know cluster formation was already using this pattern, it was internal. By keeping it internal, we can improve the implementation without needing to change 20+ projects to have a different syntax.

Reply:
I agree and don't like it either, I was planning to address it in a follow up.